remc2/engine/GameRender.cpp

```diff
@@ -14856,7 +14856,7 @@ void GameRender::WaitForRenderFinish()
 		taskCount = 0;
 		for (i = 0; i < m_renderThreads.size(); i++)
 		{
-			taskCount += m_renderThreads[i]->GetNumberRunningTasks();
+			taskCount += m_renderThreads[i]->GetIsTaskRunning();
```

Comment:
This is an implicit cast from bool to int. Better something like
```
bool finished;
do {
  finished = true;
  for (auto& thread: m_renderThreads) {
    finished &= !thread.GetIsTaskRunning();
  }
while (!finished);
```

Reply:
I didn't spot that, nice work